docs/concepts/vega-protocol.md

```diff
@@ -2,69 +2,273 @@
+
 :::info
-You'll need a Vega Wallet for staking and receiving rewards. Connect to wallets and see your account balance on the [Vega token website](https://token.vega.xyz). CoinList custodial users should confirm with CoinList how staking works for them.
+A liquidity commitment is optional when proposing a market, but a market will not enter into continuous trading until its liquidity needs are met.
+:::
+
+To propose a market, you'll need to provide the details required for the market to begin trading right away. While some of the fields are free-text, others are constrained by a range set through network parameters, to ensure that the values provided are fit for purpose.
+
+Required fields include: 
+* Instrument details, including a human-readable name, an understandable shortcode for the market, the type of product (futures)
+* Risk model parameters
+* Product specifics including the settlement asset and quote name
+* Decimal places for the settlement asset, market, and positions 
+* Oracle details, including the oracle's public key, specifications for the settlement price and trading termination, and data filters
+* Liquidity monitoring parameters, including the target stake parameters, triggering ratio and auction extension
+
+Optional fields include: 
+* Metadata so that people can easily interpret the market's details
+* Price monitoring parameters, including the triggers covering the horizon, probability and auction extension time. If left blank these parameters will default to the values set in the network parameters
+* Liquidity commitment: the amount committed, proposed fee level, and the buy and sell order shapes. Note: Once a market is proposed, it can accept liquidity commitments from any party
+
+Read more:
+* [Risk models and parameters](#risk-models-and-parameters)
+* [New market proposal](../tutorials/proposals/new-market-proposal.md): Guide to submitting a proposal for a new market using the command line
+
+<!--
+* [Data sources]
+* [Liquidity monitoring parameters]
+* [Price monitoring parameters]
+-->
+
+### Risk models and parameters
+When proposing a market, the market proposer will need to choose the risk parameters associated with the risk model that's appropriate for the instrument. The risk model is essential for calculating margins on the market. 
+
+The first product available to create is cash-settled futures, which uses a log-normal risk model. While the model is pre-defined, you'll need to choose the individual parameters.
+
+You should choose parameters that ensure the risk model adequately represents the dynamics of the underlying instrument, and that the resulting margins strike the right balance between prudence and capital efficiency.
```

Comment:
We should ask David or Witold to create a guide for how to work these out and perhaps a spreadsheet template and/or some Python script to help (and link it/them here)